wpilibc/src/main/native/include/frc/romi/RomiGyro.h

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <hal/SimDevice.h>
+
+#include "frc/GyroBase.h"
+
+namespace frc {
+
+/**
+ * Use a rate gyro to return the robots heading relative to a starting position.
+ *
+ * This class is for the Romi onboard gyro, and will only work in simulation/Romi
+ * mode. Only one instance of a RomiGyro is supported.
+ */
+class RomiGyro : public GyroBase {
+ public:
+  RomiGyro();
+
+  /**
+   * Return the actual angle in degrees that the robot is vurrently facing.
```

Comment:
vurrently -> currently

Reply:
I think we're gonna need `./gradlew spelchek` soon 😛